examples/ping_pong/src/ping_pong_sup.erl

```diff
@@ -16,8 +16,8 @@ start_link() ->
     supervisor:start_link(?MODULE, {}).
 
 start_listeners() ->
-    Port = 8080,
-    ListenerCount = 1,
+    Port          = application:get_env(nss, http_port, 8080),
```

Comment:
why have you changed this? `nss` is not an app that will run here, so it will end up as 8080 (and next line as 1) always. And it's an example app, so it really doesn't matter. Please undo this change.